docs/customizations/custom-css.md

```diff
@@ -22,6 +22,25 @@ After entering your rules, click on the "Submit" button to save your changes. Yo
 
 ![homarr custom css result](./img/customizations-custom-css-result.png)
 
+### More custom CSS rules
+
+:::info 
+
+Here is a example for a blurred background image
+
+:::
+
+![homarr custom css results](./img/customizations-custom-css-webkit.png)
```

Comment:
Very bad practice to post code as an image - it can't be copied and sucks to update.
Please use the code blocks instead. Check the Docusaurus documentation if you don't know how.